src/main/java/org/nebula/contrib/ngbatis/io/MapperResourceLoader.java

```diff
@@ -164,6 +158,7 @@ private Map<String, MethodModel> parseMethodModel(Class namespace, List<Node> no
     for (Node methodNode : nodes) {
       if (methodNode instanceof Element) {
         MethodModel methodModel = parseMethodModel(methodNode);
+        addSpaceToSessionPool(methodModel.getSpace());

```

Comment:
@soul-gin 
Space can declared in xml method tag also,
so I refact this method and move to SessionDispatcher.